Dockerfile

```
@@ -1,10 +1,7 @@
-FROM node:25-slim AS base
-ENV PNPM_HOME="/pnpm"
-ENV PATH="$PNPM_HOME:$PATH"
-RUN apt update && apt install --yes wget && rm -rf /var/lib/apt/lists/*
-RUN wget -qO- https://get.pnpm.io/install.sh | ENV="$HOME/.bashrc" SHELL="$(which bash)" bash -
-COPY . /app
+FROM ghcr.io/pnpm/pnpm AS base
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)